server/accessors/appointmentAccessor.class.php

```diff
@@ -57,7 +57,7 @@ private function resetFieldsInServicedAppointmentByAppointmentId($appointmentId)
 
 		$query = "UPDATE ServicedAppointment
 			SET timeIn = NULL, timeReturnedPapers = NULL, timeAppointmentStarted = NULL, timeAppointmentEnded = NULL,
-				completed = FALSE, cancelled = FALSE, servicedByStation = NULL
+				completed = NULL, cancelled = FALSE, servicedByStation = NULL
```

Comment:
What's the reason for this change?

Reply:
Good question--I think it was wrong before. This field starts out `null` in the DB, and we use this field to determine if an appointment is completed or not completed (obviously) where `TRUE` means completed successfully, `FALSE` means marked as incomplete, and `NULL` means isn't in a state where it's complete or incomplete. So I think `NULL` makes more sense here since this method literally just resets the fields to what they were at the start (when we first inserted the ServicedAppointment entry)